.github/workflows/notify_team_new_comment.yml

```diff
@@ -25,5 +30,5 @@ jobs:
         with:
           payload: |
             {
-              "text": "*New comment on issue: <${{ github.event.issue.html_url }}#issuecomment-${{ github.event.comment.id }}|${{ github.event.issue.title }} by ${{ github.event.comment.user.login }}>*"
+              "text": "*[Kolibri] New comment on issue: <${{ github.event.issue.html_url }}#issuecomment-${{ github.event.comment.id }}|${{ steps.escape_title.outputs.ISSUE_TITLE }} by ${{ github.event.comment.user.login }}>*"
```

Comment:
That won't work because `toJSON` outputs a string with double quotes in it, so for example -- 

Doing `toJSON("this is an issue title with double quotes " inside it")`  ---> `"this is an issue title with double quotes \" inside it"`.

So `toJSON` outputs a string with double quotes, putting that inside the payload is not going to work.

Reply:
Ah, gotcha. Irritating!